pushmsixscript/src/pushmsixscript/script.py

```diff
@@ -12,14 +12,15 @@
 
 
 async def async_main(config, task_dict):
-    msix_file_path = artifacts.get_msix_file_path(config, task_dict)
-    log.info(f"found msix at {msix_file_path}")
-    manifest.verify_msix(msix_file_path)
+    msix_file_paths = artifacts.get_msix_file_paths(config, task_dict)
+    for msix_file_path in msix_file_paths:
+        log.info(f"found msix at {msix_file_path}")
+        manifest.verify_msix(msix_file_path)
```

Comment:
If this takes a while to run, we could look at running this async as well as the push.

Reply:
This part is quite fast.